README.md

```diff
@@ -3,10 +3,56 @@
 Tools for automatically debugging dependent tests.
 Currently a work in progress.
 
+## Quickstart
+
+### Use Pre-Compiled Script
+```bash
+bash ./run.sh PROJECT_REPO_URL PROJECT_MODULE_PATH
+```
+
+### Self-Compile Scripts
```

Comment:
This instructions are quite long, so probably should not be here (especially because they're basic a copy of the script).